Search/JumpSearch.js

```diff
@@ -29,7 +29,10 @@ const jumpSearch = (arr, value) => {
   }
   return -1
 }
-const arr = [0, 0, 4, 7, 10, 23, 34, 40, 55, 68, 77, 90]
-jumpSearch(arr, 4)
-jumpSearch(arr, 34)
-jumpSearch(arr, 77)
+
+export { jumpSearch }
+
+// const arr = [0, 0, 4, 7, 10, 23, 34, 40, 55, 68, 77, 90]
```

Comment:
remove this outcommented code

Reply:
If I remove "export" then, tests will fail

Reply:
I told you to remove the outcommented code, not the `export`